BunqSdk/Exception/ApiException.cs

```diff
@@ -1,32 +1,22 @@
-﻿using System.Collections.Generic;
-
+﻿
 namespace Bunq.Sdk.Exception
 {
-    /// <summary>
-    /// Exception triggered by API requests failed on the server side.
-    /// </summary>
     public class ApiException : System.Exception
     {
-        /// <summary>
-        /// Glue to concatenate the error messages.
-        /// </summary>
-        private const string GLUE_ERROR_MESSAGES = "\n";
-
+        private readonly string message;
         public int ResponseCode { get; private set; }
-        public IList<string> Messages { get; private set; }
 
-        /// <param name="responseCode">The HTTP Response code of the failed request.</param>
-        /// <param name="messages">The list of messages related to this exception.</param>
-        public ApiException(int responseCode, IList<string> messages) : base(
-            ConcatenateMessages(messages))
+        public override string Message
         {
-            ResponseCode = responseCode;
-            Messages = messages;
+            get { return message; }
         }
 
-        private static string ConcatenateMessages(IEnumerable<string> messages)
+        /// <param name="responseCode">The HTTP Response code of the failed request.</param>
+        /// <param name="message">The list of messages related to this exception.</param>
+        public ApiException(int responseCode, string message) : base(message)
         {
-            return string.Join(GLUE_ERROR_MESSAGES, messages);
+            ResponseCode = responseCode;
+            this.message = message;
```

Comment:
Please use property here!
```
Message = message
```

Reply:
The property `Message` has no setter tho 🤔 therefore I've created 
```
        private readonly string message;
```

which will be returned instead. 🦆

Reply:
@OGKevin but you are calling the base constructor, which exactly sets this property! Therefore, I still don't wee why we would need the overridden property at all :P